filter.go

```diff
@@ -188,34 +188,6 @@ func PathDepth(depth int) Filter {
 	})
 }
 
-// value filters
-
-func Value(value string) Filter {
-	return FilterFunc(func(cookie *Cookie) bool {
-		return cookie != nil && cookie.Value == value
-	})
-}
-func ValueContains(substr string) Filter {
-	return FilterFunc(func(cookie *Cookie) bool {
-		return cookie != nil && strings.Contains(cookie.Value, substr)
-	})
-}
-func ValueHasPrefix(prefix string) Filter {
-	return FilterFunc(func(cookie *Cookie) bool {
-		return cookie != nil && strings.HasPrefix(cookie.Value, prefix)
-	})
-}
-func ValueHasSuffix(suffix string) Filter {
-	return FilterFunc(func(cookie *Cookie) bool {
-		return cookie != nil && strings.HasSuffix(cookie.Value, suffix)
-	})
-}
-func ValueLen(length int) Filter {
-	return FilterFunc(func(cookie *Cookie) bool {
-		return cookie != nil && len(cookie.Value) == length
-	})
-}
-
```

Comment:
Please remove the changes to filtering for now (this is an API change). I will try to add that delayed decryption without the removal of those filters (through some DelayedFilter type and type assertion or similar).

Reply:
If you don't think it's unreasonable, I can revise it based on the master of the current project. I'm really sorry, this part of the modification was submitted to another project.